helium/src/test/groovy/com/stanfy/helium/handler/codegen/java/entity/AndroidParcelableWriterSpec.groovy

```diff
@@ -401,6 +401,121 @@ class MyMsg {
   }
 
 
+  def "write base class calls"() {
+    given:
+    Message baseMsg = new Message(name: "Base")
+    baseMsg.addField(new Field(name: "data", type: new Type(name: "int32")))
+
+    Message msg = new Message(name: "Message", parent: baseMsg)
+    msg.addField(new Field(name: "optional", type: new Type(name: "bool")))
+
+    when:
+    // check if everything is written
+    def converter = new MessageToJavaClass(writer, options)
+    converter.write(baseMsg)
+    // reset writer
+    writer = new AndroidParcelableWriter(new PojoWriter(output), options)
```

Comment:
One writer instance does not allow to call ```write(message)``` twice, throwing ```IllegalStateException``` . Mb there is another way ?